hack/templates/aws.managed.openshift.io_v1alpha1_configmap.tmpl

```diff
@@ -20,4 +20,26 @@ objects:
     base: ${BASE}
     quota.vcpu: ${VCPU_QUOTA}
     account-limit: ${ACCOUNTLIMIT}
-    sts-jump-role: ${STS_JUMP_ARN}
+    regions: |
+      us-east-1:ami-000db10762d0c4c05:t2.micro
+      us-east-2:ami-094720ddca649952f:t2.micro
+      us-west-1:ami-04642fc8fca1e8e67:t2.micro
+      us-west-2:ami-0a7e1ebfee7a4570e:t2.micro
+      ca-central-1:ami-06ca3c0058d0275b3:t2.micro
+      eu-north-1:ami-08bc26bf92a90ba04:t3.micro
+      eu-central-1:ami-09de4a4c670389e4b:t2.micro
+      eu-west-1:ami-0202869bdd0fc8c75:t2.micro
+      eu-west-2:ami-0188c0c5eddd2d032:t2.micro
+      eu-west-3:ami-0c4224e392ec4e440:t2.micro
+      eu-south-1:ami-04684e5a51afd7579:t3.micro
+      ap-northeast-1:ami-00b95502a4d51a07e:t2.micro
+      ap-northeast-2:ami-041b16ca28f036753:t2.micro
+      ap-northeast-3:ami-070dd2ec8c4a6df38:t2.micro
+      ap-east-1:ami-f4fab885:t3.micro
+      ap-south-1:ami-0963937a03c01ecd4:t2.micro
+      ap-southeast-1:ami-055c55112e25b1f1f:t2.micro
+      ap-southeast-2:ami-036b423b657376f5b:t2.micro
+      sa-east-1:ami-05c1c16cac05a7c0b:t2.micro
+      af-south-1:ami-0f4b49fefef9be45a:t3.micro
+      me-south-1:ami-0b41a37a62a4296fc:t3.micro
```

Comment:
This is the only one that needs to be taken care of/decided on before we merge.

Reply:
@iamkirkbater You mean to https://github.com/openshift/aws-account-operator/blob/123eecb5c59940db997530a45ff05aeeaf0c8355/hack/olm-registry/olm-artifacts-template.yaml#L66-L72 ?

I'm personally fine with it residing here with the operator (as it was before when it was embedded in go), rather than in app-interface unless there is a strong reason we should move it

Reply:
Yeah, either there or we define it here in the operator.  The way it's written right now it's _expecting_ it in the configmap, but because it's not in the OLM Template it's not going to actually be in the configmap.  

We should be able to hard-code it in that file, though.